src/main/java/at/hannibal2/skyhanni/features/misc/GhostCounter.kt

```diff
@@ -54,7 +54,7 @@ object GhostCounter {
     val hidden get() = ProfileStorageData.profileSpecific?.ghostCounter
     private var display = listOf<List<Any>>()
     private var ghostCounterV3File = File("." + File.separator + "config" + File.separator + "ChatTriggers" + File.separator + "modules" + File.separator + "GhostCounterV3" + File.separator + ".persistantData.json")
-    private val skillXPPattern = ".*§3\\+(?<gained>.*).*\\((?<total>.*)\\/(?<current>.*)\\).*".toPattern()
+    private val skillXPPattern = ".*§3\\+(?<gained>.*)( .* | )\\((?<total>.*)\\/(?<current>.*)\\).*".toPattern()
```

Comment:
Why `( .* | )`?
Isnt `( .*) ` the same?

Reply:
Oh wait nvm

Reply:
You dont know how long i spent thinking that and having the regex not work in regex101